dev/docs/builtins-reference.md

```diff
@@ -384,6 +453,21 @@ y = X %*% rand(rows = ncol(X), cols = 1)
 w = lm(X = X, y = y)
 ress = slicefinder(X = X,W = w, Y = y,  k = 5, paq = 1, S = 2);
 ```
+<<<<<<< HEAD:docs/builtins-reference.md
```

Comment:
This article could help resolve this error: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

After that try the previous steps again from scratch.
1. Remove `mice` branch: `git branch -D mice`
2. Repeat the previous steps.

Reply:
cc @stanp7

Reply:
@j143 , I'm not able to resolve my issue. Further, error is occurring while removing mice branch.

Reply:
If you are on `mice` branch, deleting with `git branch -D mice` is equivalent to this image.

![image](https://user-images.githubusercontent.com/53068787/83934879-7f104e80-a7d1-11ea-9f56-d09e77b32e15.png)

So, first check out to master branch with `git checkout master`! (stay on the trunk of the tree, now cut the branch which is safer.)
:)

Reply:
Now I'm able to remove mice branch. Thanks!!
But there's one more problem. This command is not working  `git push --set-upstream-to=p****/mice-doc`.